src/LanguageServer/Impl/LanguageServer.Configuration.cs

```diff
@@ -67,11 +68,50 @@ public sealed partial class LanguageServer {
                 HandleUserConfiguredPathsChanges(userConfiguredPaths);
                 HandlePathWatchChanges(GetSetting(analysis, "watchSearchPaths", true));
                 HandleDiagnosticsChanges(pythonSection, settings);
+                HandleCodeActionsChanges(pythonSection);
 
                 _server.DidChangeConfiguration(new DidChangeConfigurationParams { settings = settings }, cancellationToken);
             }
         }
 
+        private void HandleCodeActionsChanges(JToken pythonSection) {
+            var refactoring = new Dictionary<string, object>();
+            var quickfix = new Dictionary<string, object>();
+
+            var refactoringToken = pythonSection["refactoring"];
+            var quickfixToken = pythonSection["quickfix"];
```

Comment:
`quickfix` or `quickFix`? Where are these settings coming from?

Reply:
it is coming from settings.json.

ex) "python.quickfix.addimport": false

no UI. just a way to pass setting

Reply:
Right, I was just wondering if @Anapo14 had looked at the naming ahead of time.

Reply:
Per this: https://github.com/microsoft/vscode/blob/434662c6cb11b4cad41bd4bf80c23812aeb319d6/src/vs/editor/contrib/codeAction/types.ts#L13-L19

quickfix is probably fine. I just wanted to ensure we had a standardized name.